### DISCUSSION TITLE
How to stream to the Squeezbox (Slimproto) Server.

### DISCUSSION THREAD

**Discussion by fibergm** [music-assistant/support#2825] [Q&A] [answered]
I think I read the full documentation on the Squeezbox, but I cant warp my head around it. 
Setup:
- MA installed on  HA Supervised 
- 5 Raspberrys with Squeezlite 
- MA with Squeezbox provider enabled

All of the devices work great so I rather not mess up a running system, for the sake of trying it out. 
I have figured out that with the Squeezebox provider you add like a small (I guess) squeezebox server to HA.
Also there was the mentioning of an Squeezebox Server addon but, the MA Documentation said that this could lead to undesired problems. 
Question :
How do I stream to the Squeezebox sever from for example Windows? (So using Windows as Source) 
What is the difference between the addon and the provider?

Thank you very much in advance. Solving this would really level up my SmartHome!

**Comment by OzGav** [comment, marked as answer]
Sorry you cant use Windows as a source unless whatever you are using can stream via URL. If so then you can add the URL as a manual radio station

![](https://music-assistant.io/assets/MA_banner.png)

In the picture above the ADDON is in the middle (the server). The providers allow connection to the server. On the left are all the music providers and on the right are all the player providers
